builds/jenkins-build-pack-website/Jenkinsfile

```diff
@@ -99,17 +100,29 @@ node()  {
 	gitCommitHash = scmModule.getRepoCommitHash()
 	gitCommitOwner = scmModule.getRepoCommitterInfo(gitCommitHash)
 
-	def requestId = utilModule.generateRequestId()
-	if (requestId != null) {
-		events.setRequestId(requestId)
-		environmentDeploymentMetadata.setRequestId(requestId)
-	} else {
-		error "Request Id Generation failed"
-	}
+	def requestId 
+  	if(request_id == 'none') {
+      requestId = utilModule.generateRequestId()
+      if (requestId != null) {
+          events.setRequestId(requestId)
+          environmentDeploymentMetadata.setRequestId(requestId)
+      } else {
+          error "Request Id Generation failed"
+      }
+    }
+    else {
+     echo "Skipped re-create deplyoment with r_id" + request_id
+     requestId = request_id
+    }
 
 	dir(repo_name)
 	{
-		events.sendStartedEvent('CREATE_DEPLOYMENT', 'Website deployment started', environmentDeploymentMetadata.generateDeploymentMap("started", environment, , gitCommitHash), environment)
+		 if(request_id == 'none') {
```

Comment:
This can be included in the above condition

Reply:
i don't have idea about this ... can we change that ??

Reply:
You have done similar update in other 2 jenkins-build-pack files.